dlp/snippets/deid.py

```diff
@@ -937,6 +937,79 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
+    """Uses the Data Loss Prevention API to de-identify sensitive data in a
+      string by matching against custom word list.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        input_str: The string to deidentify (will be treated as text).
```

Comment:
is this consistent with the other samples ... seems like we normally call it content_to_inspect or content_to_deid?

Reply:
other deidentified sample uses "input_str" example: deidentify_with_mask, deidentify_with_redact.